iac-extensions/arm/src/main/java/org/sonar/iac/arm/parser/bicep/BicepGrammar.java

```diff
@@ -363,6 +369,48 @@ public ObjectTypeProperty OBJECT_TYPE_PROPERTY() {
         STRING_LITERAL()));
   }
 
+  public Expression ARRAY_EXPRESSION() {
+    return b.<ArrayExpression>nonterminal(BicepLexicalGrammar.ARRAY_EXPRESSION).is(
+      f.arrayExpression(
+        b.token(Punctuator.LBRACKET),
+        b.optional(b.token(EOL)),
+        b.zeroOrMore(
+          f.tuple(
+            EXPRESSION(),
+            b.token(EOL))),
+        b.token(Punctuator.RBRACKET)));
+  }
```

Comment:
However, keeping in mind that we are planning to support comma-separated arrays as well as EOL-separated, I think it would make sense to leave it as is.